src/sentry/models/group.py

```diff
@@ -415,6 +417,7 @@ class Group(Model):
             (GroupStatus.UNRESOLVED, _("Unresolved")),
             (GroupStatus.RESOLVED, _("Resolved")),
             (GroupStatus.IGNORED, _("Ignored")),
+            (GroupStatus.ARCHIVED_UNTIL_ESCALATING, _("Archived until Escalating")),
```

Comment:
```suggestion
            (GroupStatus.ARCHIVED_UNTIL_ESCALATING, _("Archived Until Escalating")),
```

or

```suggestion
            (GroupStatus.ARCHIVED_UNTIL_ESCALATING, _("Archived until escalating")),
```

I don't think "until" is like "to" or "a" - a helper word that doesn't get capitalized in a title.